cmd/scollector/collectors/azureeabilling.go

```diff
@@ -0,0 +1,192 @@
+package collectors
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/mhenderson-so/azure-ea-billing"
+
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
+)
+
+var azureeabillingconfig = azureEABillingConfig{}
```

Comment:
Also since it is used a a lot I would make it shorter. AzBillConf maybe .. or something better